pytensor/tensor/blas.py

```diff
@@ -1880,76 +1873,26 @@ def contiguous(var, ndim):
             )
         contiguate = "\n".join(contiguate)
 
-        def c_dimshuffle(newname, oldname, shape):
```

Comment:
Why were you able to chop all this out without replacing it? It is because the logic it handled will now be re-written before it ever ends up as C code?

Reply:
Yes, I was able to chop it by making the Op only support 3d tensors as inputs and doing the dimhuffles and squeezes manually by the helper functions that calls this Op.

Since the C implementation added the dimhuffles in all cases it looked like an easy way to simplify C code